src/Game/AI/AI/aiDragonItemRoot.cpp

```diff
@@ -39,4 +43,47 @@ void DragonItemRoot::loadParams_() {
     getAITreeVariable(&mIsInsideObserverArea_a, "IsInsideObserverArea");
 }
 
+inline float sqXYZDistance(const sead::Vector3f& a, const sead::Vector3f& b) {
+    sead::Vector3f diff = a;
+    diff -= b;
+    return diff.squaredLength();
+}
+
+// Swapped arguments in fadd for distance calcuation
+#ifdef NON_MATCHING
```

Comment:
we use simple NON_MATCHING comments now:

`// NON_MATCHING: Swapped arguments in fadd for distance calcuation`

(because the ifdefs require more effort to maintain and they make it easier for the code within to bit rot)